src/models/global.rs

```diff
@@ -70,12 +70,22 @@ impl Default for FramegrabberType {
 #[derive(Debug, Clone, PartialEq, Serialize, Deserialize, Validate)]
 #[serde(default, rename_all = "camelCase", deny_unknown_fields)]
 pub struct Framegrabber {
+    pub enable: bool,
     #[serde(rename = "type")]
     pub ty: FramegrabberType,
+    #[serde(rename = "available_devices")]
+    pub available_devices: String,
+    pub device: String,
+    #[serde(rename = "device_inputs")]
+    pub device_inputs: String,
     #[validate(range(min = 10))]
     pub width: u32,
     #[validate(range(min = 10))]
     pub height: u32,
+    pub fps: u32,
+    pub framerates: String,
```

Comment:
This looks like it's a list of framerates, maybe it should be a `Vec<u32>`? Do we have a spec/schema for this somewhere?